types/setting.go

```diff
@@ -59,4 +63,22 @@ var (
 		Required:    true,
 		ReadOnly:    true,
 	}
+
+	SettingDefinitionStorageOverProvisioningPercentage = SettingDefinition{
+		DisplayName: "Storage Over Provisioning Percentage",
+		Description: "The setting used for multiple disk scheduling, set the upper bound of disks",
```

Comment:
`The over-provisioning percentage defines how much storage can be allocated relative to the hard drive's capacity`